File: book-order-app/src/app/courses/models/book.ts
//TODO: Decide what  attributes make up a book (rename these) 

export class Book {

    public bookId: string; //Unique id for each book possibly
    public bookTitle: string;
    public bookAuthors: string;
    public bookISBN: string;
    public bookPublisher: string;
    public bookEdition: string;
    public bookRequired: boolean;

    constructor(){
    }
}

File: BookOrder.Api/BookOrder.Core/Models/Book.cs
namespace BookOrder.Core.Models
{
    using System.Collections.Generic;
    public class Book
    {
        //TODO other fields as needed
        public string BookId { get; set; }
        public string BookTitle { get; set; }
        public string BookAuthors { get; set ;}
        public string BookISBN { get; set; }
        public string BookPublisher { get; set; }
        public string BookEdition { get; set; }
        public bool BookRequired { get; set; }

        public Book()
        {
        }
    }
}
